server/xpub-server/health.js

```diff
@@ -0,0 +1,35 @@
+const config = require('config')
+const { db } = require('pubsweet-server')
+const AWS = require('aws-sdk')
+
+const S3_API_VERSION = '2006-03-01'
+
+const s3 = new AWS.S3({
+  ...config.aws.credentials,
+  ...config.aws.s3,
+  apiVersion: S3_API_VERSION,
+})
+
+const checkS3 = () =>
+  new Promise(resolve => {
+    s3.listObjects({}, (err, response) => {
+      if (err) {
+        resolve(err)
+      } else {
+        resolve(response)
+      }
+    })
+  })
+
+const checkDataBase = () =>
+  new Promise(resolve => {
+    db.select('table_name')
+      .from('information_schema.tables')
+      .then(response => resolve(response))
+      .catch(error => resolve(error))
+  })
```

Comment:
We should add a unit test for this me thinks.

Reply:
What would it be testing? :/

Reply:
It's more to highlight what the result of `checkDataBase` would be, but I think you're right, it's too postgres specific.